Makefile

```
@@ -344,3 +346,8 @@ docker_security_spiffe_token_provider: docker_base
 
 vendor:
 	$(GO) mod vendor
+
+sbom:
+	docker run -it --rm \
+		-v "$(CUR_DIR):/edgex-go" -v "$(CUR_DIR)/sbom:/sbom" \
```

I think you can use `$$PWD` rather than create a separate variable:

```suggestion
		-v "$$PWD:/edgex-go" -v "$$PWD/sbom:/sbom" \
```

```Makefile
CUR_DIR=$(shell pwd)

foo:
        echo $$PWD
        echo $(CUR_DIR)
```

```sh
$ make foo
echo $PWD
/home/eojeda/make-test
echo /home/eojeda/make-test
/home/eojeda/make-test
```

---

Hi @ernestojeda thanks for the suggestion, but that doesn't seem working for me. when you use $$PWD it becomes $PWD literally instead of the value of `$PWD`

---

never mind, i got it wrong. sorry. I'll use your suggestion 👍

---

updated.